driver-core/src/test/java/com/datastax/driver/core/HostConnectionPoolTest.java

```diff
@@ -290,7 +290,7 @@ public void should_adjust_connection_keyspace_on_dequeue_if_pool_state_is_differ
             int count = 0;
             for (MockRequest queuedRequest : queuedRequests) {
                 try {
-                    Uninterruptibles.getUninterruptibly(queuedRequest.connectionFuture, 5, TimeUnit.SECONDS);
+                    Uninterruptibles.getUninterruptibly(queuedRequest.connectionFuture, 10, TimeUnit.SECONDS);
```

Comment:
Just for context for anyone else looking, this was needed because of [JAVA-1371](https://datastax-oss.atlassian.net/browse/JAVA-1371).  Previously an exception would be thrown nearly right away, after 1371 the future now fails after 5 seconds.  With only waiting up to 5 seconds, this creates a very small window for waiting to timeout before the future fails.